### PR TITLE
fix subnet private access typo

### PR DESCRIPTION
**Pull request by xingao267**
https://github.com/terraform-google-modules/terraform-google-network

### CODE REVIEW

File: templates/tfengine/components/resources/compute_networks/main.tf [hunk 1]
```diff
@@ -31,7 +31,7 @@ module "{{$resource_name}}" {
       subnet_ip              = "{{.ip_range}}"
       subnet_region          = "{{get . "compute_region" $.compute_region}}"
       subnet_flow_logs       = true
-      subnets_private_access = true
+      subnet_private_access = true
```
Comment (umairidris) [inline templates/tfengine/components/resources/compute_networks/main.tf:34]: Isn't this supposed to be {{get . subnet_private_access false}} ?

File: examples/tfengine/generated/team/live/example-prod-networks/resources/main.tf [hunk 2]
```diff
@@ -58,18 +58,18 @@ module "example_network" {
   project_id   = var.project_id
   subnets = [
     {
-      subnet_name            = "example-bastion-subnet"
-      subnet_ip              = "10.1.0.0/16"
-      subnet_region          = "us-central1"
-      subnet_flow_logs       = true
-      subnets_private_access = true
```
Comment (umairidris) [inline examples/tfengine/generated/team/live/example-prod-networks/resources/main.tf:65]: Did the schema check miss this?

File: templates/tfengine/components/resources/compute_networks/main.tf [hunk 3]
```diff
@@ -31,7 +31,7 @@ module "{{$resource_name}}" {
       subnet_ip              = "{{.ip_range}}"
       subnet_region          = "{{get . "compute_region" $.compute_region}}"
       subnet_flow_logs       = true
```
Comment (umairidris) [inline templates/tfengine/components/resources/compute_networks/main.tf:33]: Hmm.. even subnet_flow_logs is always true so the value in the config seems to be ignored

File: examples/tfengine/generated/team/live/example-prod-networks/resources/main.tf [hunk 4]
```diff
@@ -58,18 +58,18 @@ module "example_network" {
...
+      subnet_name           = "example-bastion-subnet"
+      subnet_ip             = "10.1.0.0/16"
+      subnet_region         = "us-central1"
+      subnet_flow_logs      = true
+      subnet_private_access = true
```
Comment (umairidris) [inline examples/tfengine/generated/team/live/example-prod-networks/resources/main.tf:65]: Do you know why this missed our test? Shouldn't terraform plan fail?

Comment (umairidris) [reply examples/tfengine/generated/team/live/example-prod-networks/resources/main.tf:65]: Nevermind, subnets are implemented as a map:https://github.com/terraform-google-modules/terraform-google-network/blob/master/variables.tf#L38

Comment (xingao267) [reply examples/tfengine/generated/team/live/example-prod-networks/resources/main.tf:65]: Yea, I think once the named object is there, we can make it more restrict as well in the CFT module.

**Pull request merged**